### DISCUSSION TITLE
No map on STYTJ02YM (Mi Robot Vacuum-Mop P)

### DISCUSSION THREAD

**Discussion by 0k00l** [Hypfer/Valetudo#2164] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, just started

### Your questions

Hi, i don't have map displayed on my valetudo-rooted vaccum. Space mapping works fine because robots works as expected but no map on "Map Tab". Could you tell me what i should do?
Some info about robot:
![obraz](https://github.com/user-attachments/assets/43012d55-b721-4f96-951b-eef7b0b48215)

Map tab:
![obraz](https://github.com/user-attachments/assets/5ea4aaee-cda3-4cbc-b680-58a96b02687f)

**Comment by Hypfer** [comment, marked as answer]
Make sure to configure WiFi using Valetudo as documented in the docs on valetudo.cloud
